tests/wamr-test-suites/spec-test-script/all.py

```diff
@@ -245,7 +245,7 @@ def test_case(
 
             if p.returncode:
                 print(f"failed with a non-zero return code {p.returncode}")
-                if not verbose_flag:
+                if verbose_flag:
```

Comment:
this change doesn't seem to make sense to me.

my reading of the original code is:
* if verbose_flag is true, all output is printed above.
* otherwise, it's printed only on failure.